docs/src/misc/notation.md

```diff
@@ -27,7 +27,7 @@ Within the documented functions, the utf8 symbols are used whenever possible, as
 | ``\gamma`` | a geodesic | ``\gamma_{p;q}``, ``\gamma_{p,X}`` | connecting two points ``p,q`` or starting in ``p`` with velocity ``X``. |
 | ``∇ f(p)`` | gradient of function ``f \colon \mathcal{M} \to \mathbb{R}`` at ``p \in \mathcal{M}`` | | |
 | ``\circ`` | a group operation | |
-| ``\cdot^\mathrm{H}`` | Hermitian or conjugate transposed| |
+| ``\cdot^\mathrm{H}`` | Hermitian, quaternionic or conjugate transposed| |
```

Comment:
this might be misleading, since Hermitian and conjugate are the same thing, while quaternionic is something different (we can keep the `^H` for sure, just the text is misleading
```suggestion
| ``\cdot^\mathrm{H}`` | Hermitian or conjugate transposed for both complex or quaternion matrices| |
```